resotocore/resotocore/cli/command.py

```diff
@@ -4608,6 +4608,123 @@ async def create_certificate(
             return CLISource.single(lambda: stream.just(self.rendered_help(ctx)))
 
 
+class InfrastructureAppsCommand(CLICommand):
+    """
+    ```shell
+    apps search [pattern] [someengineering/resoto-apps]
+    app info <app_name> [someengineering/resoto-apps]
+    app install <app_name> [someengineering/resoto-apps]
```

Comment:
I would opt to define the repository as parameter. `--repo foo/bla`.
This parameter defaults to `someengineering/resoto-apps`.

Reply:
Good idea! Will update the design doc.